conn.go

```diff
@@ -335,11 +335,11 @@ func (c *Conn) Close() error {
 	return err
 }
 
-// RemoteCertificate exposes the remote certificate
-func (c *Conn) RemoteCertificate() [][]byte {
+// ConnectionState returns basic DTLS details about the connection.
```

Comment:
Could you add a comment to indicate that `Export()` in v1 has been replaced by this?

Reply:
Fixed in the latest version.